sunkit_image/flct/flct.py

```diff
@@ -0,0 +1,499 @@
+    array1 : `numpy.ndarray`
+        The first array whose order is to be changed.
+    array2 : `numpy.ndarray`
+        The second array whose order is to be changed.
+    array3 : `numpy.ndarray`
+        The third array whose order is to be changed.
+
+    Returns
+    -------
+    `tuple`
+        It returns the two input arrays after changing their order from column major to
+        row major.
+    """
+
+    # Checking whether the C extension is correctly built.
+    if _pyflct is None:
+        raise ImportError("C extension for flct is missing, please rebuild.")
+
+    one, two, three = _pyflct.swap_order_three(array1, array2, array3)
+
+    return (one, two, three)
+
+
+def flct(
+    image1,
+    image2,
+    order,
+    deltat,
+    deltas,
+    sigma,
+    quiet=False,
+    biascor=False,
+    thresh=0.0,
+    absflag=False,
+    skip=None,
+    poff=0,
+    qoff=0,
+    interp=False,
+    kr=None,
+    pc=False,
+    latmin=0,
+    latmax=0.2,
+):
+    """
+    A Python wrapper which call the FLCT C routines to perform Fourier Local Correlation
+    Tracking between two images.
+
+    .. note::
+
+        * In the references there are some dat files which can be used to test the FLCT code. The
+          best method to read those dat files is the `sunkit_image.flct.read_2_images` and
+          `sunkit_image.flct.read_3_images` as the arrays would automatically be read in row major
+          format.
+        * If you use the IDL IO routines to get the input arrays from ``dat`` files,
+          the IDL routines always read the binary files in the column major, but both Python and C,
+          on which these functions are written are row major so the order of the arrays have to be
+          changed which can be done with the ``order`` keyword. This may lead to different values
+          in both the cases.
+        * If your input arrays are column major then pass the `order` parameter as `column` and it
+          will automatically take care of the order change. But this can produce some changes in
+          the values of the arrays.
+
+    Parameters
+    ----------
+    image1 : `numpy.ndarray`
+        The first image of the sequence of two images on which the procedure is to be performed.
+    image2 : `numpy.ndarray`
+        The second image of the sequence of two images taken after ``deltat`` time of the first one.
+    order : {"row" | "column"}
+        The order in which the array elements are stored that is whether they are stored as row
+        major or column major.
+    deltat : `float`
+        The time interval between the two images.
+    deltas : `float`
+        Units of length of the side of a single pixel. Velocity is computed in units of deltas/deltat.
+    sigma : `float`
+        The width of Gaussian kernel with which the images are to be modulated. Sub-images are weighted
+        by Gaussian of width sigma. If sigma is ``0`` then the overall shift between the images is
+        returned.
```

Comment:
I would replace the entire string with the one from the paper.

Sub-images are weighted by Gaussian of width sigma. If sigma
is set to 0, only single values of shifts are returned as vx, vy. These values
correspond to the overall shifts between the two images, and these values
can be used to co-register the images if desired.